.github/workflows/health_check.yml

```
@@ -1,29 +1,50 @@
+          - python-version: "3.9"
+            tox-env-py: "39"
+          - python-version: "3.10"
+            tox-env-py: "310"
+          - python-version: "3.11"
+            tox-env-py: "311"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+      - name: Installing Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)